tests/extensions/python_test_files/group_failures_3.pyt

```diff
@@ -1,7 +0,0 @@
-"""
-Test group with first item out of order
```

Comment:
Does this mean that group annotations can be out of order now?  Regardless, Shouldn't we retain this test case?

Reply:
It does mean that, yep. The tests in `group_ordering_2.pyt` test this now, so it was just redundant.